assets/sass/style.scss

```diff
@@ -3,9 +3,11 @@
 @import "../../node_modules/bulma/sass/utilities/initial-variables";
 @import "../../node_modules/bulma/sass/utilities/_all";
 @import "../../node_modules/bulma/sass/base/_all";
+@import "../../node_modules/bulma/sass/grid/_all";
```

Comment:
This doesn't seems to be used & removing it doesn't seems to change anything to the page.

Reply:
Yes I was trying to use `level` from Bulma. I think I removed all levels.

Reply:
I still can see `level` reference in the HTML tho.